src/box/sql/func.c

```diff
@@ -1102,6 +1102,22 @@ func_unicode(struct sql_context *ctx, int argc, const struct Mem *argv)
 	mem_set_uint(ctx->pOut, (uint64_t)c);
```

Comment:
In the commit message you said *"If this function is called more than once in a query, it returns
the same result until the query is completed."* - it won't be true if the request yields between `NOW()` calls. Because `ev_now()` propagates on yield. It is not a bug, but the doc needs to be fixed.

Reply:
Thanks, fixed.